datalad_crawler/nodes/tests/test_annex.py

```diff
@@ -211,22 +210,12 @@ def _test_add_archive_content_tar(direct, repo_path):
     assert_equal(output_addarchive,
                  [{'datalad_stats': ActivityStats(add_annex=1, add_git=1, files=3, renamed=2),
                    'filename': '1.tar'}])
-    if not direct:  # Notimplemented otherwise
-        assert_true(annex.repo.dirty)
+    assert_true(annex.repo.dirty)
```

Comment:
Hmph, I suppose that "not implemented" comment is stale?  If it were true, I'd guess this test will fail in the `DATALAD_REPO_DIRECT=yes` case.

Reply:
I guess it is/was ;)